roads.mss

```diff
@@ -2961,6 +2970,7 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       [highway = 'raceway'] { text-halo-fill: @raceway-fill; }
       [highway = 'service'] { text-halo-fill: @service-fill; }
       text-face-name: @book-fonts;
+      text-repeat-distance: @text-repeat-distance;
```

Comment:
This code is not working properly in my tests - see my previous comment on `text-repeat-distance` values.